intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -128,11 +129,15 @@ We might say...
 
 ### State and Behavior
 
-When we start defining our own classes, we will specify their **state** and their generic **behavior.**
+When we start defining our own classes, we will specify their **state** and their generic **behavior**.
+
+**State**, **Attribute**, **Behavior**, and **Method** are all new terms we haven't heard before, but don't be intimidated by them! 
+
+**State** is just a term that means "all the **attributes** of an object". **Attribute** is really just a specific type of **variable** defined within a class. 
 
-**State** describes any data or attributes that should be kept over its lifetime—from the moment we create an instance until we let the system reclaim its resources.
+These **attributes** will be kept over the object's lifetime—-from the moment we create an instance until the object goes out of scope.
```

Comment:
Do we talk in-depth about garbage collection ahead of this lesson? 
If we do, great! I'll put it back like it was. If we don't, I believe my change helps use the concept of scope that we've already taught them (at least in the pre-course, if not in core) rather than using concepts like reclaiming memory that require a more deep understanding than I'd expect them to have.

Reply:
We don't talk about garbage collection in-depth by this point, but we talk about objects in memory (in reasonable depth, as in, we use the phrase "objects in memory") by this point.

I won't die on this hill/I trust what you eventually decided to merge, but *pushes glasses up\* it just comes across as inaccurate/not accurate "enough" to me as a programmer that I couldn't not comment on it.

Reply:
I guess I don't understand why talking about it in terms of scope is inaccurate. That is a totally new concept/argument to me. 
I still feel pretty strongly that talking about it in terms of scope will be much more clear to students, but I'm interested to understand why that is inaccurate.

Reply:
I don't really understand what you're trying to solve with the small edit you're now suggesting either. It also feels less clear to me

Reply:
idk, I think what you said makes sense, it's good to go with your suggestion